seungjong/app.js

```diff
@@ -37,10 +36,22 @@ app.get('/ping', (req, res, next) => {
     res.status(200).json({message: "pong!!"});
 });
 
-
-const port = process.env.PORT;
+app.post('/signUp', async (req, res, next) => {
+    const {name, email, password, profile_image } = req.body;
+
+    await dataSource.query(
+        `INSERT INTO users(
+            name,
+            email,
+            password,
+            profile_image
+        ) VALUES (?, ?, ?, ?);
+        `,
+        [name, email, password, profile_image]
+    );
+    res.status(201).json({ message: "sucessfully created!" });
+})
```

Comment:
```suggestion
app.post('/signUp', async (req, res, next) => {
    const {name, email, password, profileImage } = req.body;

    await dataSource.query(
        `INSERT INTO users(
            name,
            email,
            password,
            profile_image
        ) VALUES (?, ?, ?, ?);
        `,
        [name, email, password, profileImage]
    );
    res.status(201).json({ message: "sucessfully created!" });
})
```
자바스크립트에서는 snake_case(밑줄로서 단어를 잇는)가 아닌 camelCase(소문자로 시작하여 대문자를 섞는 방법)를 사용하는것이 컨벤션입니다.